Cargo.toml

```diff
@@ -32,6 +32,9 @@ k8s-sync = { version = "0.2.3", optional = true }
 #k8s-sync = { path = "../rs-k8s-sync", optional = true }
 hyper = { version = "0.14", features = ["full"], optional = true }
 tokio = { version = "1", features = ["full"], optional = true}
+elasticsearch = "8.5.0-alpha.1"
+# bug on hyper side, socket2 needs this feature to compile
```

Comment:
Any link to this issue ?

Reply:
https://github.com/hyperium/hyper/issues/3045

Reply:
thx